packages/sds/src/message_channel/mem_local_history.ts

```diff
@@ -58,6 +43,7 @@ const log = new Logger("sds:local-history");
 
 export class MemLocalHistory implements ILocalHistory {
   private items: ContentMessage[] = [];
+  private messageIndex: Map<string, ContentMessage> = new Map();
```

Comment:
```suggestion
private orderedMessageArray: ContentMessage[] = [];
  private messageMap: Map<string, ContentMessage> = new Map();
```

I don't think a `Map` qualify as an index because it cotnains both keys (index) and values (not a ref)